pom.xml

```diff
@@ -20,47 +20,47 @@
     <repositories>
         <repository>
             <id>papermc</id>
-            <url>https://papermc.io/repo/repository/maven-public/</url>
-        </repository>
-        <repository>
-            <id>minecraft-repo</id>
-            <url>https://libraries.minecraft.net/</url>
```

Comment:
The mojang repo was necessary for the authlib dependency. Where is this pulled from now?

Reply:
Papers repo